README.md

```diff
@@ -2,6 +2,11 @@
 
 **fuels-ts** is a library for interacting with **Fuel v2**.
 
+[![build](https://github.com/FuelLabs/fuels-ts/actions/workflows/ci.yaml/badge.svg)](https://github.com/FuelLabs/fuels-ts/actions/workflows/ci.yaml)
+[![npm](https://img.shields.io/npm/v/@fuel-ts/contract)](https://www.npmjs.com/package/@fuel-ts/contract)
```

Comment:
I think this should point to `fuels` and not `@fuel-ts/contract`.

Reply:
Ok I will upgrade to this package https://www.npmjs.com/package/fuels